homeassistant/components/binary_sensor/sleepiq.py

```diff
@@ -0,0 +1,71 @@
+"""
+Support for SleepIQ sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sleepiq/
+"""
+from homeassistant.components import sleepiq
+from homeassistant.components.binary_sensor import BinarySensorDevice
+
+DEPENDENCIES = ['sleepiq']
+ICON = 'mdi:sleep'
+
+
+# pylint: disable=too-many-instance-attributes
+class IsInBedBinarySensor(sleepiq.SleepIQSensor, BinarySensorDevice):
+    """Implementation of a SleepIQ presence sensor."""
+
+    def __init__(self, sleepiq_data, bed_id, side):
+        """Initialize the sensor."""
+        sleepiq.SleepIQSensor.__init__(self,
+                                       sleepiq_data,
+                                       bed_id,
+                                       side)
+        self.type = sleepiq.IS_IN_BED
+        self._name = sleepiq.SENSOR_TYPES[self.type]
+        self.update()
+
+    @property
+    def is_on(self):
+        """Return the status of the sensor."""
+        return self._state is True
+
+    @property
+    def state(self):
```

Comment:
> Show on BinarySensorDevice to show how state work.

I don't follow what you mean here.

I already have a `is_on` in this class. My concern is that if something does check the state on this, it would get `SleepIQSensor#state` instead of `BinarySensorDevice#state`. That would return `"True"` and `"False"` rather than `"on"` and `"off"`.